integration_tests/bindc_02.py

```diff
@@ -0,0 +1,13 @@
+from ltypes import c_p_pointer, CPtr, pointer, i16
+
+# queries: CPtr
+# x: pointer(i16[:])
+# c_p_pointer(queries, x)
+# print(queries, pointer(x))
+
+def f():
+    yptr: pointer(i16[:])
+    y: i16[2]
+    y[0] = 1
+    y[1] = 2
+    y <= yptr
```

Comment:
Not sure which syntax to use the above, `y <= yptr` or `yptr = pointer(y)` to allow assigning arrays to pointers in Python. Thoughts @certik ?

Reply:
In Fortran:
```fortran
yptr => y
```
In C++:
```c++
yptr = y // (or &y if `y` was not already a pointer)
```

Reply:
So it seems `yptr = pointer(y)` should be transformed to `Associate(yptr, y)` ASR node. If on the RHS is anything else than just `pointer(y)`, then we can give an error message for now.

Reply:
Done. `yptr = pointer(y)` works now.